main.cpp

```diff
@@ -52,85 +52,85 @@ arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
 
 //  1)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    dog
+//  action 1:    the dog barks
+dog.bark();
+//  action 2:    the dog whimpers
+dog.whimper();
+//  action 3:    the dog  runs
+dog.run();
 //  2)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    engine
+//  action 1:    engine accelerates
+engine.accelerate();
+//  action 2:    engine decelerates
+engine.decelerate();
+//  action 3:    engine neutral
+engine.neutral();
 //  3)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    turntable
+//  action 1:     turntable spin at 45
+turntable.spinAtFortyFive();
+//  action 2:    turntable spin at 33
+turntable.spinAtThirtyThree();
+//  action 3:    turntable spin at 78
+turntable.spinAtSeventyEight();
 //  4)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    eyes
+//  action 1:    eyes look left
+eyes.lookLeft();
+//  action 2:    eyes look right
+eyes.lookRight();
+//  action 3:  eyes roll  
+eyes.roll();
 //  5)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    tongue
+//  action 1: tongues protrudes
+tongue.protrude();
+//  action 2:    tongue licks lips
+tongue.lickLips();
+//  action 3:    tongue touches roof of mouth
+tongue.touchRoofOfMouth();
 //  6)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    thermostat
+//  action 1:    thermostat switches on
+thermostat.switchOn();
+//  action 2:    thermostat switches off
+thermostat.switchOff();
+//  action 3:    thermostat resets
+thermostat.reset();
 //  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    hair
+//  action 1: hair grows
+hair.grow();
+//  action 2: hair greys
+hair.changeToGrey();
+//  action 3:    hair curls
+hair.curl();
 //  8)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    cat
+//  action 1:    cat leaps
+cat.leaps();
+//  action 2:    cat purrs
+cat.purr();
+//  action 3:    cat blinks    
+cat.blink();
 //  9)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    kettle
+//  action 1:    kettle consumes electricity
+kettle.consumeElectricity();
+//  action 2:    kettle heats water
+kettle.heatWater();
+//  action 3:    kettle switch illuminates
+kettle.switchIlluminate();
 //  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  Noun:    human
+//  action 1:    human laughs
+human.laugh();
+//  action 2:    human cries
+human.cry();
+//  action 3:    human gasps
+human.gasp();

```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.

Reply:
Created new noun and actions as requested.